packages/codemirror/style/index.css

```diff
@@ -25,7 +25,6 @@
   font-size: var(--jp-code-font-size);
   font-family: var(--jp-code-font-family);
   height: auto;
-  background: transparent;
```

Comment:
I reverted CSS changes in this PR because Cameron is working on the styling elsewhere.  This change was a transitory.